### ISSUE TITLE
[Re] A circuit model of auditory cortex

### ISSUE THREAD

**Issue by ChristophMetzner** [ReScience/submissions#69]
**Original article:**
Park, Y., & Geffen, M. N. (2020). A circuit model of auditory cortex. PLoS Computational Biology, 16(7), e1008016.

**PDF URL:**
https://github.com/ChristophMetzner/ReScience_ReplicationParkGeffen/blob/main/Article/article.pdf
**Metadata URL:**
https://github.com/ChristophMetzner/ReScience_ReplicationParkGeffen/blob/main/Article/metadata.yaml
**Code URL:**
https://github.com/ChristophMetzner/ReScience_ReplicationParkGeffen/tree/main/Code

**Scientific domain:**
Computational Neuroscience
**Programming language:**
Python
**Suggested editor:**
@eroesch , @otizonaizit

**Comment by rougier** [comment]
Thansk for your submission, we'll assign an editor soon.

**Comment by rougier** [comment]
@eroesch , @otizonaizit Can any of you edit this submission?

**Comment by ChristophMetzner** [comment]
Dear @rougier , this seems to have been idle for a while now. Could you assign an editor shortly please. Thank you! Christoph

**Comment by eroesch** [comment]
Hi @ChristophMetzner ! Sorry, I missed that. @rougier , I can edit it.

**Comment by ChristophMetzner** [comment]
Thanks @eroesch !

**Comment by eroesch** [comment]
@MehdiKhamassi, @rossant, @FedericoV, @gokceneraslan, @heplesser, @vitay .. Can you review, this please?

**Comment by eroesch** [comment]
.. or @otizonaizit of course! :D

**Comment by MehdiKhamassi** [comment]
Dear @eroesch ,
Thank you for the invitation. Unfortunately, I am too busy with several other reviews these days. So I prefer to decline.
Best wishes
mehdi

**Comment by rougier** [comment]
@ChristophMetzner Really really sorry for the super long delay and thank your for your patience.
For some reason I did not see it.

**Comment by ChristophMetzner** [comment]
@rougier No worries! :)

**Comment by rougier** [comment]
@eroesch Any progress on finding reviewers?

**Comment by eroesch** [comment]
@rougier None so far. I'll end up doing one I think. You interested?

**Comment by rougier** [comment]
Yes, count me in!

**Comment by eroesch** [comment]
Awesome! What is your timeline?
It's Easter holiday here. I'll need a few weeks to get it through myself.

**Comment by rougier** [comment]
End of April ?

**Comment by rougier** [comment]
Any update ?

**Comment by eroesch** [comment]
@rougier ... so have you had a go at reviewing? I am almost done with mine.

**Comment by rougier** [comment]
Nope, I totally forgot. Will do as soon as I can. Make sure to spam me !

**Comment by rougier** [comment]
We're almost 4 years late... @eroesch What's the status ?
